packages/dotcom-ui-header/src/__stories__/story.tsx

```diff
@@ -56,48 +56,6 @@ DefaultHeaderWithDrawer.args = {
   showAskButton: false
 }
 
-export const DefaultHeaderWithDrawerEntryTestAnon = (args) => (
```

Comment:
**Reviewer notes:** When we added the AB test we added lots of storybook examples to make it easy to check with and without the test variant. These are now redundant to the storybook stories become simplified again